enable-vcp-image/enable-vcp-scripts/manager_pod.sh

```diff
@@ -146,3 +149,19 @@ if [ $? -eq 0 ]; then
 else
     echo "[ERROR] 'kubectl create' failed to create vcp-daemonset."
 fi
+
+init_VcpConfigSummaryStatus "$NUMBER_OF_REGISTERED_NODES"
+
+while true
+do
+    error=$(kubectl get VcpStatus --namespace=vmware 2>&1 >/dev/null)
+    if [ -z "$error" ]; then
+        update_VcpConfigSummaryStatus "$NUMBER_OF_REGISTERED_NODES"
+        TOTAL_WITH_COMPLETE_STATUS=`kubectl get VcpStatus --namespace=vmware -o json | jq '.items[] .spec.status' | grep "${DAEMONSET_PHASE_COMPLETE}" | wc -l`
+        if [ $TOTAL_WITH_COMPLETE_STATUS -eq $NUMBER_OF_REGISTERED_NODES ]; then
```

Comment:
Can you initialize NUMBER_OF_REGISTERED_NODES right before the update logic. So that if a node goes down the failure effect it minimal.

Reply:
Initialized NUMBER_OF_REGISTERED_NODES in the very beginning, right before any phase of enabling vSphere cloud provider is executed. if nodes goes down in between, printed a log message for user, which can be viewed from manager-pods's log.

`echo "[INFO] Waiting for [${NUMBER_OF_REGISTERED_NODES}] nodes to report successfully configured. Found [${TOTAL_WITH_COMPLETE_STATUS}] nodes configured successfully."`